lesson0.md

```diff
@@ -80,14 +126,14 @@ Content by Peter.
     6.  Choose to show only 3 posts per page and allow comments on posts, but comments should be approved by an administrator before being displayed.
     7.  Find a Copyright-free image with an appropriate size and use it as the site's header.
     8.  Add a menu with the following items: Home, About and Contact. "Home" should link to the main blog page and "Contact" should link to an email address.
-    
-    Do as many of the tasks below as you can but don't worry if you get stuck with any of the steps. There is a lot of information in the [WordPress.com support site](https://en.support.wordpress.com/) or, if you can't find a solution there, please ask on Slack.
-    
+
+    Do as many of the tasks below as you can but don't worry if you get stuck with any of the steps. There is a lot of information in the [WordPress.com support site](https://en.support.wordpress.com/) or, if you can't find a solution there, please don't hesitate to ask us on Slack.
```

Comment:
Another guide that I found useful in customer/client support is https://easywpguide.com/. It just gives very simple screenshots for basic tasks like adding a page and choosing a theme.